client/README.md

```diff
@@ -1,17 +1,30 @@
 # Galaxy Tools (Visual Studio Code Extension)
+
 This extension provides XML validation, tags and attributes completion, help/documentation on hover, and other *smart* features to assist in following best practices during the development process of XML tool wrappers for [Galaxy](https://galaxyproject.org/).
 
 > Please note this is still a work in progress so **bugs and issues are expected**. If you find any, you are welcome to open a new [issue](https://github.com/galaxyproject/galaxy-language-server/issues).
 
-# Requires ![PyPI - Python Version](https://img.shields.io/pypi/pyversions/galaxy-language-server)
-In order to use the [Galaxy Language Server](https://pypi.org/project/galaxy-language-server/) features you need Python 3.8+ installed on your system. See the [Installation](#Installation) section for more details.
+# Requirements
+
+## Requires ![PyPI - Python Version](https://img.shields.io/pypi/pyversions/galaxy-language-server)
+
+To use the [Galaxy Language Server](https://pypi.org/project/galaxy-language-server/) features you need Python 3.8+ installed on your system. See the [Installation](#Installation) section for more details.
+
+## Planemo
+
+Since version `0.4.0` you can use some of the cool features of [planemo](https://github.com/galaxyproject/planemo) directly from the extension. You only need to provide the required path to your planemo installation (see [configuration](#planemo-settings)) and the necessary parameters.
```

Comment:
can the extension by default just assume that planemo in on PATH?

Reply:
Good idea! done! ;)